cmd/ipfs-cluster-service/main.go

```diff
@@ -189,10 +189,10 @@ func main() {
 			Name:  "debug, d",
 			Usage: "enable full debug logging (very verbose)",
 		},
-		cli.StringFlag{
-			Name:  "loglevel, l",
-			Value: defaultLogLevel,
-			Usage: "set the loglevel for cluster components only [critical, error, warning, info, debug]",
+		cli.StringSliceFlag{
```

Comment:
Let's make this a StringFlag and parse the `--loglevel` value in the same way the environment value is parsed. A format which needs to be explained.